docs/cookbook/recipe_workflow_integration.rst

```diff
@@ -67,14 +67,14 @@ You can use the provided extension to take care of your entity admin.
 
        app.admin.extension.workflow.blog_post:
            class: Yokai\SonataWorkflow\Admin\Extension\WorkflowExtension
-           public: true
            arguments:
                - '@workflow.registry'
                - transitions_icons:
                      start_review: fa fa-question
                      interrupt_review: fa fa-edit
                      restart_review: fa fa-question
                      publish: fa fa-check
+           public: true
```

Comment:
why?

Reply:
Also, does it really need to be public?

Reply:
You are right, currently we generate them with public true, we should change this too

Reply:
Fixed the whole admin-bundle docs in #5377